requirements.txt

```diff
@@ -0,0 +1,4 @@
+
+python==3.9.13
+django==4.0
+djangorestframework
```

Comment:
Why is there not a version?

Reply:
Because we can use the latest version

Reply:
I checked in the [Django website](https://www.django-rest-framework.org/#requirements)